pl_examples/basic_examples/conv_sequential_example.py

```diff
@@ -222,6 +222,6 @@ def instantiate_datamodule(args):
     trainer.fit(model, cifar10_dm)
     trainer.test(model, datamodule=cifar10_dm)
 
-    if trainer.accelerator_backend.rpc_enabled:
+    if trainer.accelerator.rpc_enabled:
```

Comment:
just a side note, wont this confuse user with Trainer argument which is string?

Reply:
good point, I have not thought about this. 
On the other hand, we never store the string input anywhere, so the user has to go through distrib_type.
@justusschock what is your preference?

Reply:
Hey both, I think `accelerator` is better. Most users might not use the trainer directly and even less the accelerator.

However, we should add a depreciation there for `accelerator_backend` property @awaelchli.

Reply:
I also think we should rename it and add a property linking to it for the old name with raising a depreciation warning as @tchaton mentioned.

Reply:
ok, will add that warning